doc/swss-schema.md

```diff
@@ -620,7 +622,42 @@ Equivalent RedisDB entry:
     12) "0"
     127.0.0.1:6379>
 
-### Configuration files
+
+## Configuration DB schema
+
+### WARM\_RESTART
+    ;Stores system warm start configuration
+    ;Status: work in progress
+
+    key             = WARM_RESTART:name         ; name is the name of SONiC docker or "system" for global configuration.
+
+    enable          = "true" / "false"          ; Default value as false.
+                                                ; If "system" warm start knob is true, docker level knob will be ignored.
+                                                ; If "system" warm start knob is false, docker level knob takes effect.
```

Comment:
After discussion with @zhenggen-xu ,  a little more clarification about the use cases for system knob and docker knobs.
<1> Upon system reboot, the system enable knob will be checked.  If enabled, database data will be preserved, if not, database will be flushed.
No need to check docker level knobs in this case since the whole system is being rebooted .

<2> Upon docker service start,  first to check system knob, if enabled, docker warm start should be performed, otherwise system warm reboot will be ruined.  If system knob disabled, while docker knob enabled, this is likely an individual docker warm restart request. 

Within each application, when the system knob or docker knob enabled, we do further check on the actual warm start state ( restart_count),  if no warm start state data available, the database has been flushed, do cold start. Otherwise warm start.

The exact logic has been implemented in WarmStart::checkWarmStart():
https://github.com/Azure/sonic-swss/pull/547/files

Reply:
I am fine with this logic.